packages/runtime/container-runtime/src/deltaScheduler.ts

```diff
@@ -86,7 +90,18 @@ export class DeltaScheduler {
 
                 // eslint-disable-next-line @typescript-eslint/no-floating-promises
                 this.deltaManager.inbound.pause();
+
                 setTimeout(() => {
+                    if (this.schedulingLog) {
+                        this.logger.sendTelemetryEvent({
+                            eventName: "DeltaManagerPaused",
+                            duration: elapsedTime,
+                            numberOfOps: this.deltaManager.inbound.length,
```

Comment:
I'd rename numberOfOps to something like opsRemainingToProcess
And for consistency - numberofOpsProcessed -> opsProcessed